lib/eval-snippets.ts

```diff
@@ -114,6 +114,12 @@ export const snippets = {
         }) || true,
     EVAL_ETSY_1: () => document.querySelector('.gdpr-overlay-view button[data-wt-overlay-close]').click() || true,
     EVAL_EZOIC_0: () => ezCMP.handleAcceptAllClick(),
+    EVAL_FASTCMP_OPTIN: () =>
+        document?.querySelector?.('iframe#fast-cmp-iframe')?.contentDocument?.querySelector?.('.fast-cmp-home-accept button')?.click?.() ||
```

Comment:
Could we just implement iframe piercing in the same way we do it for [shadowRoots](https://github.com/duckduckgo/autoconsent/blob/main/lib/dom-actions.ts#L134-L136)? This way we could just pass a selector array to desend into the frame document if the first selector has a `contentDocument`.

This would replace these evals with a simple step:
```json
{
  "click": ["iframe#fast-cmp-iframe", ".fast-cmp-home-refuse button"]
}
```